### ISSUE TITLE
Drag anywhere on the window

### ISSUE THREAD

**Issue by cassidyjames** [danrabbit/nimbus#21]
Right now it looks like you can only drag on the indistinguishable titlebar. It'd be rad to drag it around from anywhere on the window.

**Comment by sgpthomas** [comment]
Magic code that does this. Just add it to the constructor of the main window
```
/**
 *  Allow moving the window
 */
this.button_press_event.connect ((e) => {
		if (e.button == Gdk.BUTTON_PRIMARY) {
			this.begin_move_drag ((int) e.button, (int) e.x_root, (int) e.y_root, e.time);
			return true;
		}
		return false;
	});
```

**Comment by danrabbit** [comment]
Thanks! Fixed in fc2667d8012c200cc2b18d1291f572d63295fe00
